extensions/agroal/runtime/src/main/java/io/quarkus/agroal/runtime/AgroalEventLoggingListener.java

```diff
@@ -61,6 +61,11 @@ public void onWarning(String warning) {
         log.warnv("{0}: {1}", datasourceName, warning);
     }
 
+    @Override
+    public void onInfo(String message) {
+        log.infov("{0}: {1}", datasourceName, message);
```

Comment:
Quick question that might be stupid: so now, we will log the message about the initial size at info level? Does it really solve the problem? I mean most users will run their application with the default log level.

Reply:
> Quick question that might be stupid: so now, we will log the message about the initial size at info level?

Yes.

> Does it really solve the problem?

What problem exactly? See also discussion on #4808 

It will read like this now, wording suggested by Loic; too late to change the wording as that requires an Agroal release.

` INFO  [io.agr.pool] (main) Datasource '<default>': Initial size smaller than min. Connections will be created when necessary
`

We could make it entirely silent by setting the default min-size to zero: it's 5 now and I'm not entirely sure why.

Reply:
ah that's actually not the wording Loic suggested. Looks like it was too late for the Agroal train.